zokrates_cli/src/ops/mpc/export.rs

```diff
@@ -0,0 +1,98 @@
+use crate::constants::{MPC_DEFAULT_PATH, PROVING_KEY_DEFAULT_PATH, VERIFICATION_KEY_DEFAULT_PATH};
+use clap::{App, Arg, ArgMatches, SubCommand};
+use phase2::parameters::MPCParameters;
+use std::fs::File;
+use std::io::{BufReader, Write};
+use std::path::Path;
+use zokrates_core::proof_system::bellman::{parse_g1, parse_g2};
+use zokrates_core::proof_system::groth16::VerificationKey;
+use zokrates_field::Bn128Field;
+
+pub fn subcommand() -> App<'static, 'static> {
+    SubCommand::with_name("export")
+        .about("Exports keys from MPC parameters")
+        .display_order(5)
+        .arg(
+            Arg::with_name("input")
+                .short("i")
+                .long("input")
+                .help("Path of the MPC parameters")
+                .value_name("FILE")
+                .takes_value(true)
+                .required(false)
+                .default_value(MPC_DEFAULT_PATH),
+        )
+        .arg(
+            Arg::with_name("proving-key-path")
+                .short("p")
+                .long("proving-key-path")
+                .help("Path of the generated proving key file")
+                .value_name("FILE")
+                .takes_value(true)
+                .required(false)
+                .default_value(PROVING_KEY_DEFAULT_PATH),
+        )
+        .arg(
+            Arg::with_name("verification-key-path")
+                .short("v")
+                .long("verification-key-path")
+                .help("Path of the generated verification key file")
+                .value_name("FILE")
+                .takes_value(true)
+                .required(false)
+                .default_value(VERIFICATION_KEY_DEFAULT_PATH),
+        )
+}
+
+pub fn exec(sub_matches: &ArgMatches) -> Result<(), String> {
+    let path = Path::new(sub_matches.value_of("input").unwrap());
+    let file =
+        File::open(&path).map_err(|why| format!("Could not open `{}`: {}", path.display(), why))?;
+
+    let reader = BufReader::new(file);
+    let mpc_params = MPCParameters::read(reader, true)
+        .map_err(|why| format!("Could not read `{}`: {}", path.display(), why))?;
+
+    println!("Exporting keys from `{}`...", path.display());
+
+    let params = mpc_params.get_params();
+
+    let mut pk: Vec<u8> = Vec::new();
+    params.write(&mut pk).unwrap();
+
+    let vk = VerificationKey {
```

Comment:
We usually have this parsing code in zokrates_core so that we can access it from js or cli

Reply:
Now I realise it's the case for most of the additions in this PR, do you see anything preventing us from having a `zokrates_mpc` crate and only do I/O in the CLI? It could be a separate PR.

Reply:
yeah I was thinking about doing this at first, but wanted to have this done asap